crates/exec-wasmtime/src/loader/mod.rs

```diff
@@ -16,8 +16,8 @@ mod connected;
 mod pki;
 mod requested;
 
-use super::config::Config;
 use super::{Args, Package};
+use enarx_config::EnarxConfig;
```

Comment:
```suggestion
use enarx_config::Config;
```

Would be way nicer to use

Reply:
I knew you would say this

Reply:
fixed